kv/utils.go

```diff
@@ -18,10 +18,12 @@ import (
 	"strconv"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 )
 
 // IncInt64 increases the value for key k in kv store by step.
 func IncInt64(rm RetrieverMutator, k Key, step int64) (int64, error) {
+	rm.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlreadyFull)
```

Comment:
why  here? I think only genGlobalIDs(create table ...) needs this

Reply:
Just wants to try to let all the little data change transactions to be dealt with when TiKV is disk full.
Because such scenarios are some basic ability of other processing.

Reply:
Done